### PR TITLE
Drop styled link text

### PR DESCRIPTION
**Pull request by guardrex**

### CODE REVIEW

File: aspnetcore/mvc/models/validation.md [hunk 1]
```diff
@@ -129,7 +129,7 @@ The `AdditionalFields` property of the `[Remote]` attribute lets you validate co
 
 [!code-csharp[](validation/samples/3.x/ValidationSample/Models/User.cs?name=snippet_Name&highlight=1,5)]
 
-`AdditionalFields` could be set explicitly to the strings `"FirstName"` and `"LastName"`, but using the [`nameof`](/dotnet/csharp/language-reference/keywords/nameof) operator simplifies later refactoring. The action method for this validation must accept both `firstName` and `lastName` arguments:
+`AdditionalFields` could be set explicitly to the strings `"FirstName"` and `"LastName"`, but using the [nameof`](/dotnet/csharp/language-reference/keywords/nameof) operator simplifies later refactoring. The action method for this validation must accept both `firstName` and `lastName` arguments:
```
Comment (Rick-Anderson) [inline aspnetcore/mvc/models/validation.md]: `AdditionalFields` could be set explicitly to the strings `"FirstName"` and `"LastName"`, but using the [nameof](/dotnet/csharp/language-reference/keywords/nameof) operator simplifies later refactoring. The action method for this validation must accept both `firstName` and `lastName` arguments:

Comment (Rick-Anderson) [reply aspnetcore/mvc/models/validation.md]: Missed trailing \`

File: aspnetcore/security/authentication/identity.md [hunk 2]
```diff
@@ -368,7 +368,7 @@ Post is specified in the *Pages/Shared/_LoginPartial.cshtml*:
 
 ## Test Identity
 
-The default web project templates allow anonymous access to the home pages. To test Identity, add [`[Authorize]`](/dotnet/api/microsoft.aspnetcore.authorization.authorizeattribute) to the Privacy page.
+The default web project templates allow anonymous access to the home pages. To test Identity, add [[Authorize]](/dotnet/api/microsoft.aspnetcore.authorization.authorizeattribute) to the Privacy page.
```
Comment (Rick-Anderson) [inline aspnetcore/security/authentication/identity.md]: I think these look better code fenced. We're more flexible on that that the style guide.

File: aspnetcore/razor-pages/index.md [hunk 3]
```diff
@@ -191,7 +191,7 @@ Reviewing the *Pages/Create.cshtml* view file:
 [!code-cshtml[](index/3.0sample/RazorPagesContacts/Pages/Customers/Create.cshtml?highlight=3,9)]
 
 * In the preceding code, the [input tag helper](xref:mvc/views/working-with-forms#the-input-tag-helper) `<input asp-for="Customer.Name" />` binds the HTML `<input>` element to the `Customer.Name` model expression.
-* [`@addTagHelper`](xref:mvc/views/tag-helpers/intro#addtaghelper-makes-tag-helpers-available) makes Tag Helpers available.
+* [@addTagHelper](xref:mvc/views/tag-helpers/intro#addtaghelper-makes-tag-helpers-available) makes Tag Helpers available.
```
Comment (Rick-Anderson) [inline aspnetcore/razor-pages/index.md]: prefer original. Hopefully this renders OK

File: aspnetcore/security/authentication/identity.md (hunk 2 above)
Comment (guardrex) [reply aspnetcore/security/authentication/identity.md]: Perhaps one to two dozen of these that have brackets in link text were done this way (no styling) ... and these few that I updated here had styling. There was at least an inconsistency present within the repo.

File: aspnetcore/razor-pages/index.md (hunk 3 above)
Comment (guardrex) [reply aspnetcore/razor-pages/index.md]: Same deal here ... ~45 occurrences without styling. This is perhaps the only one **with** styling.

**Pull request merged**